cables.py

```diff
@@ -8,7 +8,7 @@
 
 
 loglevel = logging.INFO
-
+clean = re.compile('<.*?>')
```

Comment:
Why as global when it is only once used?

Reply:
it is better to compile once instead of every time the remove_html_tags method is called